Added steps for anyone who is interested our project's  collaborate task. (#241)
CodeforAustralia/council-emissions-calculator: README.md

</details>

> ## Collaboration: Want to collaborate with this project ?

Reviewer: ```suggestion
## Collaboration: Want to collaborate on this project ?
```

<details>
<summary> 
STEPS
</summary>

> 1. Forked the project repository from the owners repository.

Reviewer: ```suggestion
1. Fork the project repository from this repository.
```

Author: I have corrected  the suggested changes. Thank you

Reviewer: It still shows the original text @Amutha37 , do you want to push the changes?

Reviewer 2: Hi @Amutha37 ,

I think this conversation has gone on unresolved for too long. I'll resolve this for now, and address outstanding work in a separate PR.

   (go to your git account you will see you have a copy now)

2. Open terminal `git clone ...` your forked repository .

> 3. List the current configured remote repository for your fork.
>    `git remote -v`
> 
>    > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
>    > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
> 
> 4. Specify a new remote upstream repository that will be synced with the fork.
>    ` git remote add upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git`
> 
> 5. Verify the new upstream repository you've specified for your fork.
>    `git remote -v`
> 
>    > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (fetch)
>    > origin https://github.com/YOUR_USERNAME/YOUR_FORK.git (push)
>    > upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (fetch)
>    > upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git (push)
> 
> 6. `git checkout -b (name of the branch)` Now you can work on the branch.

Reviewer: ```suggestion
3. `git checkout -b (name of the branch)` Now you can work on the branch.
```

Hi @Amutha37 , great thought on updating the instructions. I think your steps are clear. However, should we just keep steps 1, 2 then 6 (as 3rd step)? For majority of people you never need to set up upstream as it will be set to default branch anyway (in our case `staging`). The only time you play with upstream is when you base your new branch on a different branch. I assume it might have happened to you as you used `main` branch without realising the default is now `staging`.

Author: Thank you for the feedback. Yes, I agree with you . The question is what if we at sill at 'main' branch and not in 'staging' . Like we was working initially . Is this something I should consider in the README before removing those lines that you feel not necessary.

Reviewer 2: Hi @Amutha37 ,

I think this conversation has gone on unresolved for too long. I'll resolve this for now, and address outstanding work in a separate PR.
